packages/uikit/src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -194,17 +194,16 @@ const DropdownMenu: React.FC<React.PropsWithChildren<DropdownMenuProps>> = ({
     [items, isMobile, isMd]
   );
   const hasItems = filteredItems.length > 0;
-  const { styles, attributes } = usePopper(targetRef, tooltipRef, {
+  const { styles, attributes, update } = usePopper(targetRef, tooltipRef, {
     strategy: isBottomNav ? "absolute" : "fixed",
     placement: isBottomNav ? "top" : "bottom-start",
     modifiers: [{ name: "offset", options: { offset: [0, isBottomNav ? 6 : 0] } }],
   });
 
-  const isMenuShow = isOpen && ((isBottomNav && showItemsOnMobile) || !isBottomNav);
+  const isMenuShown = isOpen && ((isBottomNav && showItemsOnMobile) || !isBottomNav);
 
   useEffect(() => {
-    if (isBottomNav && !hasItems) return undefined;
-    if (trigger !== "hover") return undefined;
+    if (!hasItems || trigger !== "hover") return undefined;
```

Comment:
isBottomNav check not needed if dropdown doesnt have any items in that case listeners shouldn't be set on page dropdowns too